.github/workflows/scripts/check-pr-description.sh

```diff
@@ -40,15 +42,21 @@ any_of=(
 )
 for task in "${any_of[@]}"; do
   if [[ "$PR_DESCRIPTION" = *"$EOL- [ ] $task"* ]]; then
-    gh_log error "'${task%% }' task is not checked."
-    exit 1
+    gh_log error "'$task' task is not checked."
+    fail="true"
   fi
 done
+
+found="false"
 for task in "${any_of[@]}"; do
   if [[ "$PR_DESCRIPTION" = *"$EOL- [x] $task"* ]]; then
-    gh_log debug "at least '$task' task is checked."
-    exit 0
+    gh_log debug "'$task' task is checked."
+    found="true"
   fi
 done
-gh_log error 'none of the automated tests tasks are checked.'
-exit 1
+
+[[ "$found" = "false" ]] && gh_log error 'none of the automated tests tasks are checked.'
```

Comment:
Is this also a failure case ?

Reply:
Yup! At least one should be checked, and there should be no unchecked.

Reply:
Then I suppose `fail` should be set in that context as well.
```
if [[ "$found" = "false" ]]; then
    gh_log error 'none of the automated tests tasks are checked.'
    fail="true"
fi
```